docs/resources/aws_albs.md

```diff
@@ -49,9 +57,9 @@ See also the [AWS documentation on Elastic Load Balancing](https://docs.aws.amaz
 
 ## Matchers
 
-This InSpec audit resource has no special matchers. For a full list of available matchers, please visit our [Universal Matchers page](https://www.inspec.io/docs/reference/matchers/).
+This InSpec audit resource has no special matchers. For a full list of available matchers, please visit our [Universal Matchers page](/inspec/matchers/).
```

Comment:
Changing this to a relative link will break it for users looking directly at the resource pack docs folder. Could this be updated to the full link?